### PR TITLE
Update lightningdesignsystem.json

### PR DESCRIPTION
**Pull request by kaelig**
This adds a little bit of hierarchy between sections of our site.

### CODE REVIEW

**Comment by ElPicador** [comment]
LGTM, @maxiloc could you also have a look?

File: lightningdesignsystem.json [hunk 1]
```diff
@@ -1,7 +1,34 @@
 {
   "index_name": "lightningdesignsystem",
   "start_urls": [
-    "https://www.lightningdesignsystem.com/"
+    {
+      "url": "https://www.lightningdesignsystem.com/",
+      "page_rank": 1
+    },
```
Comment (maxiloc) [inline lightningdesignsystem.json]: we need to put this one last. Otherwise 👍

Comment (kaelig) [inline lightningdesignsystem.json]: Thanks, I've put it last.

**Comment by maxiloc** [comment]
👍 Deploying it now

**Comment by kaelig** [comment]
Thanks!

**Pull request merged**